addons/pkg/constants/constants.go

```diff
@@ -169,6 +169,18 @@ const (
 
 	// PackageInstallSyncPeriod is the sync period for kapp-controller to periodically reconcile a PackageInstall
 	PackageInstallSyncPeriod = time.Minute * 10
+
+	// WebhookCertDir is the directory where the certificate and key are stored for webhook server TLS handshake
+	WebhookCertDir = "/tmp/k8s-webhook-server/serving-certs"
+
+	// WebhookServiceName is the name of the k8s service that serves the admission requests
+	WebhookServiceName = "webhook-service"
+
+	// WebhookScrtName is the name of secret that holds certificates and key for webhook service
+	WebhookScrtName = "webhook-tls"
+
+	// CNIWebhookLabel is the label for admission webhooks for CNI
+	CNIWebhookLabel = "cni-webhook"
```

Comment:
Question: Could we use an overall label for all the addon related webhooks?

Reply:
Fixed